crates/modelardb_compression/src/lib.rs

```diff
@@ -45,7 +45,7 @@ pub const GORILLA_MAXIMUM_LENGTH: usize = 50;
 /// Compress `uncompressed_timestamps` using a start time, end time, and a
 /// sampling interval if regular and delta-of-deltas followed by a variable
 /// length binary encoding if irregular. `uncompressed_values` is compressed
-/// within `error_bound` using the model types in [`models`](crate::models).
+/// within `error_bound` using the model types in [`models`](models).
```

Comment:
Maybe remove "(models)" as it should not be necessary [for rustdoc to create the correct link](https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html).

Reply:
Fixed.